tests/puppeteer/campaigns/sanity/02_productsBO/03_CRUDStandardProductWithCombinationsInBO.js

```diff
@@ -60,18 +64,21 @@ describe('Create, read, update and delete Standard product with combinations in
   });
 
   it('should reset all filters', async function () {
+    await testContext.addContextItem(this, 'stepIdentifier', 'resetFilters0', baseContext);
```

Comment:
```suggestion
    await testContext.addContextItem(this, 'stepIdentifier', 'resetFilters', baseContext);
```
Not sure it's needed to have the 0 index :thinking: